.github/workflows/test.yml

```diff
@@ -6,7 +6,6 @@ on:
 
 env:
   FOUNDRY_PROFILE: ci
-  FOUNDRY_VERSION: v1.0.0
```

Comment:
Let's set version to nightly here, to keep it in one place? Or maybe it's even better to put it in the CI variables?
https://docs.github.com/en/actions/learn-github-actions/variables

Reply:
Yes, I added it to the repository variables